Symfony/CS/Fixer/Contrib/PhpUnitStrictFixer.php

```diff
@@ -19,22 +19,18 @@
  */
 final class PhpUnitStrictFixer extends AbstractFixer
 {
-    private $phpUnitMethods = array(
+    private $configuration = array(
         'assertAttributeEquals' => 'assertAttributeSame',
         'assertAttributeNotEquals' => 'assertAttributeNotSame',
         'assertEquals' => 'assertSame',
         'assertNotEquals' => 'assertNotSame',
     );
 
-    public function configure(array $usingMethods = null)
+    public function configure(array $usingMethods)
     {
-        if (null === $usingMethods) {
-            return;
-        }
-
-        foreach (array_keys($this->phpUnitMethods) as $method) {
+        foreach (array_keys($this->configuration) as $method) {
             if (!in_array($method, $usingMethods, true)) {
-                unset($this->phpUnitMethods[$method]);
+                unset($this->configuration[$method]);
```

Comment:
What is the reason for the different configuration types between the two fixers?
set/not set VS. true/false